docs/content/preview/yugabyte-voyager/migrate/assess-migration.md

```diff
@@ -0,0 +1,185 @@
+---
+title: YB Voyager Migration Assessment
+linkTitle: Assess migration
+headcontent: Steps to create a migration assessment report
+description: Steps to create a migration assessment report to ensure successful migration using YugabyteDB Voyager.
+menu:
+  preview_yugabyte-voyager:
+    identifier: assess-migration
+    parent: migration-types
+    weight: 101
+techPreview: /preview/releases/versioning/#feature-availability
+type: docs
+---
+
+The Voyager Migration Assessment feature is specifically designed to optimize the database migration process from various source databases, currently supporting PostgreSQL to YugabyteDB. Voyager conducts a detailed analysis of the source database by capturing essential metadata and metrics. It generates a comprehensive assessment report that recommends effective migration strategies and provides key insights on ideal configuration strategies for optimal performance in YugabyteDB.
+
+## Overview
+
+Voyager collects various metadata or metrics from the source database, such as table columns metadata, sizes of tables/indexes, read/write IOPS for tables/indexes, and so on. With this data, an assessment report is prepared which covers the following key details:
+
+- Database compatibility: Assesses the compatibility of the source database with YugabyteDB, identifying unsupported features and data types.
```

Comment:
As far as I know, bullet points by design cannot be made bold. I made the text bold. In case you come across any docs page that has bullets in bold, please let me know.